lib/to_h.rb

```diff
@@ -39,4 +39,19 @@ def to_h
 
   end
 
-end
\ No newline at end of file
+  module Enumerable
+
+    # @return [Hash]
+    def to_h
```

Comment:
Enumerable#to_h should take arguments.

ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-linux]

```ruby
Enumerable.instance_method(:to_h).arity #=> -1
```